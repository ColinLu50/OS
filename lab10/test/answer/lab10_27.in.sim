1530
587
765
499
575
383
