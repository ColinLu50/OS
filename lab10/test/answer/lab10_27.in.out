FCFS
281 -> 287 -> 435 -> 159 -> 228 -> 450 -> 319 -> 363 -> 484 -> 308 -> 206 -> 268 -> 95
1530
SSTF
281 -> 287 -> 268 -> 228 -> 206 -> 159 -> 95 -> 308 -> 319 -> 363 -> 435 -> 450 -> 484
587
SCAN
281 -> 268 -> 228 -> 206 -> 159 -> 95 -> 0 -> 287 -> 308 -> 319 -> 363 -> 435 -> 450 -> 484
765
C-SCAN
281 -> 268 -> 228 -> 206 -> 159 -> 95 -> 0 -> 505 -> 484 -> 450 -> 435 -> 363 -> 319 -> 308 -> 287
499
LOOK
281 -> 268 -> 228 -> 206 -> 159 -> 95 -> 287 -> 308 -> 319 -> 363 -> 435 -> 450 -> 484
575
C-LOOK
281 -> 268 -> 228 -> 206 -> 159 -> 95 -> 484 -> 450 -> 435 -> 363 -> 319 -> 308 -> 287
383
